ctriface/orch.go

```diff
@@ -97,16 +99,17 @@ func NewOrchestrator(snapshotter, hostIface string, opts ...OrchestratorOption)
 	var err error
 
 	o := new(Orchestrator)
-	o.vmPool = misc.NewVMPool()
 	o.cachedImages = make(map[string]containerd.Image)
 	o.snapshotter = snapshotter
 	o.snapshotsDir = "/fccd/snapshots"
-	o.hostIface = hostIface
+	o.netPoolSize = 50
```

Comment:
This should not be hardcoded. you introduced the corresponding runtime argument, right?

Reply:
It isn't hardcoded, it is simply the default value if the client does not provide the `WithNetPoolSize` option, which will be applied later on:
https://github.com/vhive-serverless/vHive/blob/3a22b773f6f978372ca19db3e332e4b101031ae6/ctriface/orch.go#L103-L105

Reply:
set to 10 please